src/frequenz/sdk/timeseries/logical_meter/_formula_generators/_pv_power_formula.py

```diff
@@ -32,9 +37,12 @@ async def generate(self) -> FormulaEngine:
         )
 
         if not pv_inverters:
-            raise ComponentNotFound(
-                "Unable to find any PV inverters in the component graph."
-            )
+            logging.warning("Unable to find any PV inverters in the component graph.")
+            # If there are no PV inverters, we have to send 0 values as the same
+            # frequency as the other streams.  So we add a fake component, just to get a
+            # `None` message at the resampling interval.
+            await builder.push_component_metric(sys.maxsize, nones_are_zeros=True)
+            return builder.build()
```

Comment:
1. Could you make `sys.maxsize` constant?
2. Could you extend warning  info log with information that you try to subscribe to non-existing component (with component id). I am asking because it will result in misleading error logs from MicrogridApi.
3. Please change description: `So we add a fake component...` ->  `So we subscribe for not existing component...`.  It took me while to check where you add component.

Reply:
done all 3.